tools/Environments/DevHome.Environments/ViewModels/OperationsViewModel.cs

```diff
@@ -98,6 +113,38 @@ public void InvokeAction()
         });
     }
 
+    [RelayCommand]
+    public void InvokeAction()
+    {
+        // Show confirmation popup in case of delete
+        if (Name == _stringResource.GetLocalized("Operations_Delete"))

```

Comment:
Better to use string.compare than ==, but in any case, comparing against a localized string doesn't seem great. Can you compare against the ComputeSystemOperation instead? The enum seems safer.

Reply:
Good point. Changed.